terraform/modules/tf_stream_alert_cloudtrail/iam.tf

```diff
@@ -0,0 +1,45 @@
+// Role for CloudWatch events
```

Comment:
How so? This role module uses `events.amazonaws.com` as the Principal while I believe the flow log module delegates to a different service.